changelog/controller.md.gotmpl

```diff
@@ -19,4 +19,4 @@ Images:
 * {{ . }}
 {{- end }}
 {{ end }}
-**Full Changelog**: https://github.com/kubernetes/ingress-nginx/compare/controller-{{ .PreviousControllerVersion }}...controller-{{ .NewControllerVersion }}
+**Full Changelog**: https://github.com/kubernetes/ingress-nginx/compare/{{ .PreviousControllerVersion }}...{{ .NewControllerVersion }}
```

Comment:
Guess the above changelog has been created using this version, right? The Git tag already contains `controller-` then?

Reply:
yes, it was controller-controller-v1.x.x when it generated it